en/publish/custom-project-build-template.md

```diff
@@ -57,8 +57,111 @@ module.exports = {
 };
 ```
 
-The event registered above is `'before-change-files'`, which is triggered **before** the end of the build. In addition to computing the file MD5 and encryption scripts for native platform, most build operations have been completed. You can further process the files that have been built in this event.
+In the example above, we are monitoring the events of the Builder `'before-change-files'`, our `onBeforeBuildFinish` handler function is invoked when the event is triggered. The following events are currently being monitored:
```

Comment:
In the example above, we are listening the event `'before-change-files'` on the Builder, ...

Reply:
The following events are currently being supported: